.pylintrc

```diff
@@ -2,10 +2,14 @@
 ignore =
     typings
 
+[MESSAGES CONTROL]
+disable =
+    fixme,
+    line-too-long,
```

Comment:
shouldn't there be a setting so that you can be congruent with your formatter (e.g. black) ?

Reply:
Black actually allows lines of ongoing length in cases where the comments are being used to disable things. I could set this to bump up to the current longest line, but given we're not manually formatting anything (black is doing it all) I don't think there is any further benefit in having pylint running on line-length.

That was my thinking anyway...

Reply:
If it's possible to put comments in to .pylintrc to explain the reasoning for certain choices, that would be helpful to the reviewer as well as any future developers.

Reply:
Good call, I'll see if comments are allowed and make that change. I'll pull this put out into an issue for a subsequent PR. Thanks Stuart :)